content/en/docs/components/pipelines/operator-guides/installation/_index.md

```diff
@@ -1,11 +1,27 @@
 +++
 title = "Installation"
-description = "Options for deploying Kubeflow Pipelines"
 weight = 1
 +++
 
 {{% kfp-v2-keywords %}}
 
-This page will be available soon. For similar information, see [KFP v1 installation documentation][v1-installation].
+As an alternative to deploying Kubeflow Pipelines (KFP) as part of the
+[Kubeflow deployment](/docs/started/#installing-kubeflow), you also have a choice
+to deploy only Kubeflow Pipelines. Follow the instructions below to deploy
+Kubeflow Pipelines standalone using the supplied kustomize manifests.
 
-[v1-installation]: /docs/components/pipelines/legacy-v1/installation/
\ No newline at end of file
+You should be familiar with [Kubernetes](https://kubernetes.io/docs/home/),
+[kubectl](https://kubernetes.io/docs/reference/kubectl/overview/), and [kustomize](https://kustomize.io/).
+
+## Deploying Kubeflow Pipelines
+
+1. Deploy the Kubeflow Pipelines:
+
+     ```
+     export PIPELINE_VERSION={{% pipelines/latest-version %}}
```

Comment:
I think we can do it automatically by looking at `https://raw.githubusercontent.com/kubeflow/website/master/layouts/shortcodes/pipelines/latest-version.html`.

Reply:
`{{% pipelines/latest-version %}}` is replaced with the latest version during the build.
You can see that in: https://deploy-preview-3793--competent-brattain-de2d6d.netlify.app/docs/components/pipelines/operator-guides/installation/